### ISSUE TITLE
Error during gatsby server side rendering

### ISSUE THREAD

**Issue by lusarz** [auth0/lock#2061]
### Describe the problem
We are getting:
```
"window" is not available during server side rendering.
```

![Screenshot from 2021-10-27 16-50-44](https://user-images.githubusercontent.com/10059264/139090647-b75ff2e1-684d-4830-9b2f-55ea26d75d4e.png)

### Proposed solution
Wrap code that uses `window` with:
```
if (typeof window !== "undefined") {
  ...
}
```

### Environment
Gatsby

**Comment by stevehobbsdev** [comment]
Thanks for @lusarz, I have a fix for this in #2057 and looking to put out a release this week.

**Comment by stevehobbsdev** [comment]
We've tried to resolve this in [v11.31.1](https://github.com/auth0/lock/releases/tag/v11.31.1) - please try it out and let us know if this is now working for you.
